docs/concepts/07_dip/02_provider.md

```diff
@@ -0,0 +1,81 @@
+After removal, the identity becomes unusable cross-chain, although it continues to exist on the provider chain and is usable for local operations.
+
+#### Add the pallet to the runtime
+
+Add the pallet to runtime in your `Cargo.toml` file dependencies section:
+
+:::info
+
+The DIP pallet is available in the KILT node release 1.12.0 and later.
+
+:::
+
+```toml
+provider = {package = "pallet-dip-provider", git = "https://github.com/KILTprotocol/kilt-node.git", branch = "release-1.12.0"}
+```
+
+#### Configure the pallet
+
+Pass runtime configuration to the pallet's `Config` trait.
+
+```rust
+impl pallet_dip_provider::Config for Runtime {
+    // Config
+}
+```
+
+<!-- TODO: Link for full details -->
+
+The following configuration parameters are required. Read the xxx :
+
+-   `type CommitOriginCheck: EnsureOrigin<Self::RuntimeOrigin, Success = Self::CommitOrigin>`: The check ensures a given runtime origin is allowed to generate and remove identity commitments.
+-   `type CommitOrigin: SubmitterInfo<Submitter = Self::AccountId>`: The resulting origin if `CommitOriginCheck` returns with errors. The origin is not required to be an `AccountId`, but must include information about the `AccountId` of the tx submitter.
+-   `type Identifier: Parameter + MaxEncodedLen`: The type of an identifier used to retrieve identity information about a subject.
+-   `type IdentityCommitmentGenerator: IdentityCommitmentGenerator<Self>`: The type responsible for generating identity commitments, given the identity information associated with a given `Identifier`.
+-   `type IdentityProvider: IdentityProvider<Self>`: The type responsible for retrieving the information associated with a subject given their identifier. The information can potentially be retrieved from any source, using a combination of on-chain and off-chain solutions.
+-   `type IdentityProvider: IdentityProvider<Self>`: Customizable external logic to handle events in which a new identity commitment is generated or removed.
+-   `type RuntimeEvent: From<Event<Self>> + IsType<<Self as frame_system::Config>::RuntimeEvent>`: The aggregate `Event` type.
+
+#### Storage
+
+The pallet stores identity commitments, which are opaque byte blobs in the pallet storage and on which you can build the cross-chain identity bridging protocol.
+
+The runtime must define an identity commitment and is provider-specific.
+You must make this definition available to consumers willing to integrate the identities on the provider chain.
+
+The pallet contains a single storage element, the `IdentityCommitments` double map.
+Its first key is the `Identifier` of subjects, while the second key is the commitment version.
+The values are identity commitments.
+The double map allows the same subject to have one commitment for each version supported by the provider, without forcing consumers to upgrade to a new version to support the latest commitment scheme.
+
+#### Events
```

Comment:
I am wondering if there is a way to avoid duplication of docs between this place and the node repo. In there we have an MD file as well, do you think it would be best to embed that here? As long as we use fixed versions to pin the docs, it should be fine. Then we could join efforts to make those READMEs better. What do you think?